.pipelines/unit-tests-template.yml

```diff
@@ -13,6 +13,8 @@ jobs:
           version: 1.15
       - script: make build
         displayName: Build
+      - script: make lint
```

Comment:
should lint be before build so it'll fail fast?

Reply:
Done!